.gitignore

```diff
@@ -26,3 +26,6 @@ test-output/
 /server-logs/
 /integration-test-results/
 /integration-test-results.zip
+
+# ignores removed projects
+/fhir-replication-api/**/*
```

Comment:
do we need this?

Reply:
Yes - it drives me nuts when we don't - general practice is to keep open Eclipse projects from leaking back in

Reply:
ok, i don't think i agree.  i personally don't like seeing remnants of project history...prefer a clean gitignore.  i don't think we'd "leak project files back in given that all eclipse project files are ignored already.

Reply:
If you want, I'll remove it.  best practice in my opinion is to add the gitignore